src/v2/BaseLoginRouter.js

```diff
@@ -153,7 +153,27 @@ export default Router.extend({
   },
 
   handleIdxResponseFailure (error = {}) {
-    // 1. loosely check whether is IDX error response
+    // will happen if `useInteractionCodeFlow` is true but the Org is not using OIE
+    if (error?.error === 'access_denied') {
+      // simulate an IDX error response
+      error = Object.assign({
```

Comment:
We'll likely need to reuse this error creation logic. Want to move it out to a shared file?

Reply:
This is a very special case, receiving a "V1" error while operating on an IDX code path. The error says IDX is not enabled. For other errors on this path, it is expected they would be in an IDX format